wp-content/civi-extensions/civirazorpay/CRM/Core/Civirazorpay/Payment/Razorpay.php

```diff
@@ -462,6 +466,21 @@ private function processSubscriptionCharged(array $event) {
 
       $pendingContribution = $this->getPendingContributionByRecurId($recurringContribution['id']);
 
+      $sourceField = CustomField::get(FALSE)
+        ->addSelect('id')
+        ->addWhere('custom_group_id:name', '=', 'Contribution_Details')
+        ->addWhere('name', '=', 'PAN_Card_Number')
+        ->execute()->single();
+
+      $sourceFieldId = 'custom_' . $sourceField['id'];
+
+      $campaigns = Campaign::get(FALSE)
```

Comment:
Since you are finding the first one.
```suggestion
      $campaign = Campaign::get(FALSE)
```